src/main/resources/liquibase/crop_changelog/15_3_0.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="aldrin" id="v15.3.0-1">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="germplasm_study_source"/>
+			</not>
+		</preConditions>
+		<comment>
+			Create new germplasm_study_source
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			CREATE TABLE `germplasm_study_source` (
+			`source_id` INT NOT NULL AUTO_INCREMENT,
```

Comment:
I am more inclined to use germplasm_study_source_id

Reply:
Done